dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -69,6 +69,10 @@ CONFIG_ATTRIBUTE_IGNORED=SESN0311W: The {0} configuration attribute was specifie
 CONFIG_ATTRIBUTE_IGNORED.explanation=The specified configuration is ignored as it is provided by the specified cacheManagerRef.
 CONFIG_ATTRIBUTE_IGNORED.useraction=No action is required.
 
+UPDATED_CONFIG_NOT_USED_AT_RESTORE=SESN0312W: The {0} HTTP Session Cache configuration attribute changed since the server was checkpointed. If the value of the attribute is changed after the server checkpoint, but before the server process is restored, the application does not use the updated value.
+UPDATED_CONFIG_NOT_USED_AT_RESTORE.explanation=The specified configuration attribute cannot be modified during the time between the server checkpoint and the server process restore.
+UPDATED_CONFIG_NOT_USED_AT_RESTORE.useraction=To change the value for the attribute, wait until after the server process is restored. Then modify the value and create a new server checkpoint.
```

Comment:
> Then modify the value and create a new server checkpoint.

To me it sounds like this is suggesting that you take another checkpoint after the server has been restored from a previous checkpoint.  We do not support taking another checkpoint on the restored server process.

The user action is to set the value to the final desired configuration value before taking the initial checkpoint and to not change that value before restoring the server process.

Suggestion:

> Before performing a server checkpoint the value for the attribute must be set to the final value to be used when the server process is restored.

Reply:
sounds good except want to avoid elliptical clause "Before performing"- i left another suggestion.

> We do not support taking another checkpoint on the restored server process.

The thing that's confusing about this message is we're telling the user to set the value before checkpoint and not change it. But if they're seeing this message, isn't it already too late for that? They see the message because they changed the value. So how do they remedy the error now if the value set before checkpoint was incorrect?